backend/migrate_imports_dates.py

```
@@ -0,0 +1,161 @@
+    accounts = []
+    try:
+        # Use paginated scan to handle large datasets
+        table = get_accounts_table()
+        response = table.scan()
+        
+        while True:
+            for item in response.get('Items', []):
+                try:
+                    account = Account.from_dynamodb_item(item)
+                    accounts.append(account)
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)